src/SFA.DAS.EAS.Web/Controllers/EmployerAccountController.cs

```diff
@@ -25,7 +25,8 @@ public ActionResult Gateway()
         [Route("gatewayResponse")]
         public ActionResult GateWayResponse()
         {
-            return Redirect(Url.EmployerAccountsAction("gatewayResponse"));
+            return Redirect(Url.EmployerAccountsAction($"gatewayResponse?{Request.QueryString}"));

```

Comment:
I'm guessing that the Request.QueryString is already URL encoded and so doesn't need any checks of that nature.

Reply:
that is a good point. I should have used Request.url.query instead but the QueryString.tostring worked on this occasion (probably because there is only one param)

Reply:
This has now been changed to the correct value which is encoded to URI specification